README.md

```diff
@@ -172,9 +172,12 @@ You may also want to:
      before publication, or with a regular audit of a sample of documents. The
      [transforms](transformer/) are designed to work on valid AMP pages, and
      may break invalid AMP in small ways.
+  4. Setup
+     [monitoring](#monitoring-amppackager-in-production-via-its-prometheus-endpoints)
+     of `amppackager` and related requests to AMP document server.
 
 Once you've done the above, you should be able to test by launching Chrome
-without any comamndline flags; just make sure
+without any command line flags; just make sure
 chrome://flags/#enable-signed-http-exchange is enabled. To test by visiting the
```

Comment:
While you're here, mind deleting "just make sure without any command line flags; just make sure chrome://flags/#enable-signed-http-exchange is enabled"? This is really out of date. This flag no longer exists; it's launched fully in Chrome.